matcha/models/components/flow_matching.py

```diff
@@ -111,8 +118,9 @@ def compute_loss(self, x1, mask, mu, spks=None, cond=None):
         # sample noise p(x_0)
         z = torch.randn_like(x1)
 
-        y = (1 - (1 - self.sigma_min) * t) * z + t * x1
+        # y = (1 - (1 - self.sigma_min) * t) * z + t * x1
         u = x1 - (1 - self.sigma_min) * z
+        y = u * t + z
```

Comment:
I feel this would give different results. Despite of that, I still prefer the original formulations as they are identical to the flow-matching paper.